webapp/package.json

```diff
@@ -54,6 +54,9 @@
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "~2.1.0",
-    "typescript": "5.1.6"
+    "typescript": "^5.5.4"
+  },
+  "optionalDependencies": {
+    "@rollup/rollup-linux-x64-gnu": "^4.21.2"
```

Comment:
Roll up have platform specific code, add explicit `@rollup/rollup-linux-x64-gnu` for it to work in CI

Reply:
Thanks for the assist!